cmd/clusterctl/client/cluster/mover.go

```diff
@@ -76,8 +83,10 @@ func (o *objectMover) Move(namespace string, toCluster Client) error {
 	// This is required because if the infrastructure is provisioned, then we can reasonably assume that the objects we are moving are
 	// not currently waiting for long-running reconciliation loops, and so we can safely rely on the pause field on the Cluster object
 	// for blocking any further object reconciliation on the source objects.
-	if err := o.checkProvisioningCompleted(objectGraph); err != nil {
-		return err
+	if !o.isDryrun {
+		if err := o.checkProvisioningCompleted(objectGraph); err != nil {
```

Comment:
Being a dry-run, I think we should avoid also `checkTargetProviders`. WDYT?

Adding some more context, I'm wondering if we still need to access to a target cluster when dry-running or we should support dry running without providing a `--to-kubeconfig` argument

Reply:
um.. this need further check, I will consider it